src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -116,34 +144,67 @@ void EnHoll_SwapRooms(PlayState* play) {
     play->roomCtx.unk_30 ^= 1;
 }
 
-// Horizontal Planes
-void func_80A58DD4(EnHoll* this, PlayState* play) {
+/**
+ * When traversing a holl of this kind, it attempts to:
+ *   Load Current Room (fails as it is already loaded)
+ *   Load Next Room
+ *   Load Previous Room
+ *   Load Next Room
+ *
+ *  @bug The striped nature of holls can cause some actors to unload due to
+ *      conflicting Object Lists between the two rooms
+ *
+ *  @bug If you can get around to the other side of the holl without triggering it,
+ *      you can load the room on the other side multiple times
+ */
```

Comment:
Most of the comments on `EnHoll_HorizontalVisibleNarrow` come from master, it just appears as new because I moved it

in master: https://github.com/zeldaret/oot/blob/aa48c66e10fc067e4d41c3a1d097bd83dea53acf/src/overlays/actors/ovl_En_Holl/z_en_holl.c#L48-L68

I'm okay with rewording or removing some of it, but I mean I can't defend it as I didn't write them

Reply:
alright. yeah I kinda feel like its not worth mentioning at all, but if others want to keep it, it should prob just be without a bug tag in whatever place is relevant 

it shuold be relevant to more than just this type anyway, so it doesnt make sense on just this function as-is

Reply:
Actually I need to check but it may be just this type that has the "duplicate room load" bug. I will know more once room loading is documented, which is also why I left this as-is